sim/src/mechanics/driving.rs

```diff
@@ -357,6 +375,14 @@ impl DrivingSimState {
                         // Don't schedule a retry here.
                         return false;
                     }
+                    if let Some((trip, _)) = car.trip_and_person {
+                        let delay = (now - blocked_since).inner_seconds() as u8;
+                        // Maybe alter the minimum delay time to be recorded
```

Comment:
Same, the event should always be sent with the `Duration`, and the filtering + transformation to `u8` can happen later.